### PR TITLE
feat: add Boxplot viz

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: packages/viz/src/hooks/useBoxplotData.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    let tableData: ColumnTable;
+    if (data instanceof internal.ColumnTable) {
+      tableData = data;
+    } else if (Array.isArray(data)) {
+      tableData = from(data);
+    } else {
+      tableData = table(data);
+    }
+
+    const groupByField = groupBy
```
Comment (MEsteves22) [inline packages/viz/src/hooks/useBoxplotData.tsx:29]: If we can only use one value for the `groupBy` prop, then we should update `HvBoxplotProps` and only accept a `string`.

File: packages/viz/src/Boxplot/Boxplot.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+export interface HvBoxplotProps
+  extends HvAxisChartCommonProps,
+    HvChartCommonProps {
+  /** The name of the heatmap */
+  name?: string;
+  /**  Columns to measure on the chart. */
+  measures: Arrayable<HvBoxplotMeasure>;
+  /** The tooltip options. */
+  tooltip?: Omit<HvChartTooltip, "type">;
```
Comment (MEsteves22) [inline packages/viz/src/Boxplot/Boxplot.tsx:35]: Is `valueFormatter` and `titleFormatter` used? If not, we should omit these props I think.

File: packages/viz/src/types/measures.ts [hunk 3]
```diff
@@ -53,6 +53,8 @@ export interface DonutFullMeasure extends BaseMeasure {}
 
 export interface ConfusionMatrixMeasure extends BaseMeasure {}
 
+export interface BoxplotMeasure extends Omit<BaseMeasure, "agg">, AxisMeasure {}
```
Comment (MEsteves22) [inline packages/viz/src/types/measures.ts:56]: Is `valueFormatter` used? If not, I think we should omit it.

**Review by MEsteves22 — COMMENTED** [review]
👌🏻

I just have the following comments:

**Comment by plagoa** [comment]
@MEsteves22 made some changes based on your review.

File: packages/viz/src/Boxplot/Boxplot.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+export type HvBoxplotClasses = ExtractNames<typeof useClasses>;
+
+export interface HvBoxplotProps
+  extends HvAxisChartCommonProps,
+    Omit<HvChartCommonProps, "groupBy"> {
+  /** The name of the heatmap */
+  name?: string;
+  /**  Columns to measure on the chart. */
+  measures: Arrayable<HvBoxplotMeasure>;
+  /** */
```
Comment (MEsteves22) [inline packages/viz/src/Boxplot/Boxplot.tsx:34]: Missing description

File: packages/viz/src/Boxplot/useBoxplotData.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import { useMemo } from "react";
+import { from, internal, table } from "arquero";
+import type ColumnTable from "arquero/dist/types/table/column-table";
+
+import { HvChartData } from "../types";
+import { HvChartCommonProps } from "../types/common";
+
+interface HvDataHookProps {
+  data: HvChartData;
+  groupBy?: HvChartCommonProps["groupBy"];
```
Comment (MEsteves22) [inline packages/viz/src/Boxplot/useBoxplotData.tsx:10]: This is a `string` now

File: packages/viz/src/Boxplot/useBoxplotData.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    let tableData: ColumnTable;
+    if (data instanceof internal.ColumnTable) {
+      tableData = data;
+    } else if (Array.isArray(data)) {
+      tableData = from(data);
+    } else {
+      tableData = table(data);
+    }
+
+    const groupByField = groupBy
```
Comment (MEsteves22) [inline packages/viz/src/Boxplot/useBoxplotData.tsx:29]: We no longer need this condition as it will be a `string`

**Review by MEsteves22 — APPROVED** [review]
🚀 

Just found the following:

**Review by HQFOX — APPROVED** [review]

**Pull request merged**